src/backend/distributed/planner/distributed_planner.c

```diff
@@ -93,15 +93,20 @@ PlannedStmt *
 distributed_planner(Query *parse, int cursorOptions, ParamListInfo boundParams)
 {
 	PlannedStmt *result = NULL;
-	bool needsDistributedPlanning = NeedsDistributedPlanning(parse);
+	bool needsDistributedPlanning = false;
 	Query *originalQuery = NULL;
 	PlannerRestrictionContext *plannerRestrictionContext = NULL;
 	bool setPartitionedTablesInherited = false;
+	List *rangeTblList = ExtractRangeTblList(parse);
 
 	if (cursorOptions & CURSOR_OPT_FORCE_DISTRIBUTED)
 	{
 		needsDistributedPlanning = true;
 	}
+	else if (CitusHasBeenLoaded())
```

Comment:
Maybe worth a comment / assert in the if statement above to make it clear that `needsDistributedPlanning` can only be true if `CitusHasBeenLoaded()`